azurerm/internal/services/web/resource_arm_app_service_hybrid_connection.go

```diff
@@ -0,0 +1,186 @@
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringLenBetween(6, 50),
+			},
+			"relay_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"relay_arm_uri": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: azure.ValidateResourceID,
+			},
+			"hostname": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"port": {
+				Type:         schema.TypeInt,
+				Required:     true,
+				ValidateFunc: validate.PortNumberOrZero,
+			},
+			"service_bus_namespace": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"service_bus_suffix": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"send_key_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"send_key_value": {
+				Type:      schema.TypeString,
+				Required:  true,
+				Sensitive: true,
+			},
+		},
+	}
+}
+
+func resourceArmAppServiceHybridConnectionCreateUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).Web.AppServicesClient
+	ctx, cancel := timeouts.ForCreateUpdate(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	name := d.Get("app_service_name").(string)
+	resGroup := d.Get("resource_group_name").(string)
+	namespaceName := d.Get("namespace_name").(string)
+	relayName := d.Get("relay_name").(string)
+	relayNameArmUri := d.Get("relay_arm_uri").(string)
+	hostname := d.Get("hostname").(string)
+	port := int32(d.Get("port").(int))
+	serviceBusSuffix := d.Get("service_bus_suffix").(string)
+	serviceBusNamespace := d.Get("service_bus_namespace").(string)
+	sendKeyName := d.Get("send_key_name").(string)
+	sendKeyValue := d.Get("send_key_value").(string)
+
+	hybridConnectionProperties := web.HybridConnectionProperties{
+		ServiceBusNamespace: &serviceBusNamespace,
+		RelayName:           &relayName,
+		RelayArmURI:         &relayNameArmUri,
+		Hostname:            &hostname,
+		Port:                &port,
+		SendKeyName:         &sendKeyName,
+		SendKeyValue:        &sendKeyValue,
+		ServiceBusSuffix:    &serviceBusSuffix,
+	}
+
+	connectionEnvelope := web.HybridConnection{}
+	connectionEnvelope.HybridConnectionProperties = &hybridConnectionProperties
+
+	if _, err := client.CreateOrUpdateHybridConnection(ctx, resGroup, name, namespaceName, relayName, connectionEnvelope); err != nil {
+		return fmt.Errorf("Error creating App Service Hybrid Connection %q (resource group %q): %s", name, resGroup, err)
+	}
+	return resourceArmAppServiceHybridConnectionRead(d, meta)
+}
+
+func resourceArmAppServiceHybridConnectionRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).Web.AppServicesClient
+	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	id, err := azure.ParseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resGroup := id.ResourceGroup
```

Comment:
for consistency could we make this 
```suggestion
	resourceGroup := id.ResourceGroup
```